pkg/avancement/service_manager.go

```diff
@@ -251,8 +251,16 @@ func createPullRequest(ctx context.Context, fromURL, toURL, newBranchName, commi
 	}
 
 	u, _ := url.Parse(toURL)
-	// take out ".git" at the end
-	pr, _, err := client.PullRequests.Create(ctx, u.Path[1:len(u.Path)-4], prInput)
+	// take out ".git" at the end only if present
```

Comment:
I think you can use `strings.TrimSuffix(u.Path, ".git")`

Reply:
Good call, thanks, updated and tested (so both with and without the suffix) and PRs continue to look good